app/zaaktypes.py

```diff
@@ -953,6 +954,231 @@ def defer_transform(zaak_deferred, decosjoin_service):
+        },
+        {
+            "name": "fiscalnumberLocation1",
+            "from": "text18",
+            "parser": to_string,
+        },
+        {
+            "name": "urlLocation1",
+            "from": "text19",
+            "parser": to_string,
+        },
+        {
+            "name": "streetLocation2",
+            "from": "text15",
+            "parser": to_string,
+        },
+        {
+            "name": "housenumberLocation2",
+            "from": "num15",
+            "parser": to_int,
+        },
+        {
+            "name": "locationkindLocation2",
+            "from": "text20",
+            "parser": to_string,
+        },
+        {
+            "name": "fiscalnumberLocation2",
+            "from": "text21",
+            "parser": to_string,
+        },
+        {
+            "name": "urlLocation2",
+            "from": "text22",
+            "parser": to_string,
+        },
+        {
+            "name": "licensePlates",
+            "from": "text13",
+            "parser": BZP.to_kenteken,
+        },
+        {
+            "name": "previousLicensePlates",
+            "from": "text14",
+            "parser": BZP.to_kenteken,
+        },
+        {
+            "name": "dateStart",
+            "from": "date6",
+            "parser": to_date,
+        },
+        {
+            "name": "dateEnd",
+            "from": "date8",
+            "parser": to_date,
+        },
+    ]
+
+    def has_valid_source_data(self):
+        not_before = to_date("2023-08-08")
+        creation_date = to_date(self.zaak_source["document_date"])
+
+        return creation_date >= not_before and super().has_valid_payment_status()
+
+
+class EigenparkeerplaatsOpheffen(Zaak):
+    # !!!!!!!!!!!!!
+    enabled = not IS_PRODUCTION
+    # !!!!!!!!!!!!!
+
+    zaak_type = "Eigen parkeerplaats opheffen"
+    title = "Eigen parkeerplaats opheffen"
+
+    date_workflow_active_step_title = "Status bijwerken en notificatie verzenden - In behandeling"
+
+    @staticmethod
+    def defer_transform(zaak_deferred, decosjoin_service):
+        date_workflow_active = decosjoin_service.get_workflow(
+            zaak_deferred["id"], EigenparkeerplaatsOpheffen.date_workflow_active_step_title
+        )
+        zaak_deferred["dateWorkflowActive"] = date_workflow_active
+
+    parse_fields = [
+        {
+            "name": "isCarsharingpermit",
+            "from": "bol8",
+            "parser": to_bool,
+        },
+        {
+            "name": "streetLocation1",
+            "from": "text25",
+            "parser": to_string,
+        },
+        {
+            "name": "housenumberLocation1",
+            "from": "num14",
+            "parser": to_int,
+        },
+        {
+            "name": "locationkindLocation1",
+            "from": "text17",
+            "parser": to_string,
+        },
+        {
+            "name": "fiscalnumberLocation1",
+            "from": "text18",
+            "parser": to_string,
+        },
```

Comment:
Is het misschien consistent om hier ook `location: {...}` te gebruiken?

Reply:
Ik heb al die velden hernoemt, hier is namelijk altijd maar sprake van 1 locatie. Dus dit wordt nu `street`, `houseNumber` etc.